assets/scss/admin/account.scss

```diff
@@ -73,6 +73,29 @@
             }
         }
     }
+
+    .fs-button-inline
+    {
+        background-color: white;
```

Comment:
- [ ] Let' make it `background-color: transparent`. 

- [ ] And our convention to write SCSS is

```scss
.fs-button-inline {
    // ...
}
```

- [ ] Let's move this styling to a separate file for re-usability. [This file](https://github.com/Freemius/wordpress-sdk/blob/master/assets/scss/admin/_buttons.scss) seems like the correct place.